web/src/pages/OnboardPage/OnboardPage.js

```diff
@@ -21,7 +21,7 @@ const OnboardPage = () => {
   const [create, { loading }] = useMutation(ONBOARD_USER, {
     onCompleted: async () => {
       await reauthenticate()
-      navigate(routes.dashboard())
+      navigate(routes.setting())
```

Comment:
This will need to be changed once the route is renamed:

```suggestion
      navigate(routes.settings())
```